src/apollo/KeepLink.ts

```diff
@@ -53,6 +61,12 @@ export function removeIgnoreSetsFromDocument<T extends DocumentNode>(
 }
 
 export class KeepLink extends ApolloLink {
+  _enabledFeatures: Array<string> = []
+
+  setEnabledFeatures(features: Array<string>): void {
```

Comment:
We can leverage the `set` modifier here `set enabledFeatures(features:...`). See also https://www.typescripttutorial.net/typescript-tutorial/typescript-getters-setters/. Then we can simplify the access:

```ts
const keepLink = new KeepLink()
keepLink.enabledFeatures = [ 'feature1' ]
```